test/util/number_conversions.cpp

```diff
@@ -0,0 +1,38 @@
+#include <iostream>
+#include <mbgl/test/util.hpp>
+
+#include <mbgl/util/string.hpp>
+
+using namespace mbgl;
+
+TEST(NumberConversions, number_to_string) {
+    EXPECT_EQ("1", util::toString(1));
+    EXPECT_EQ("-1", util::toString(-1));
+    EXPECT_EQ("32768", util::toString(32768));
+    EXPECT_EQ("-32768", util::toString(-32768));
```

Comment:
Other things to test:

* `0` → `0`
* `-0` → `-0` (not sure about this one)
* `10` → `10`
* `-10` → `-10`